core/src/test/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpServiceFactoryTest.java

```diff
@@ -181,6 +186,70 @@ public void testCreateAnnotatedServiceElementWithoutExplicitPathOnMethod() {
         });
     }
 
+    @Test
+    public void testMultiPathSuccessGetMapping() {
+        final List<AnnotatedHttpServiceElement> getServiceElements =
+                getServiceElements(new MultiPathSuccessService(), "getMapping", HttpMethod.GET);
+        assertThat(getServiceElements).hasSize(1);
+        assertThat(getServiceElements.get(0).route().methods()).isEqualTo(ImmutableSet.of(HttpMethod.GET));
+        assertThat(getServiceElements.get(0).route().paths()).isEqualTo(ImmutableList.of("/get", "/get"));
```

Comment:
- `Route` implements `equals()`, so we could do the following for the most assertions in this test case?
   ```java
   assertThat(getServiceElements).containsOnly(
           Route.builder().get("/").build());
   ```
- Could you explain why two same paths (`/get`) are returned?

Reply:
> Route implements equals(), so we could do the following for the most assertions in this test case?

Looks much cleaner this way:) Thanks

> Could you explain why two same paths (/get) are returned?

I was thinking because of this!

https://github.com/line/armeria/blob/84d3b36239acc3276ef16baf7c002e2fc9cd8405/core/src/main/java/com/linecorp/armeria/server/ExactPathMapping.java#L40